frame/offences/src/lib.rs

```diff
@@ -110,13 +110,6 @@ pub mod pallet {
 		/// \[kind, timeslot\].
 		Offence { kind: Kind, timeslot: OpaqueTimeSlot },
 	}
-
-	#[pallet::hooks]
-	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
-		fn on_runtime_upgrade() -> Weight {
-			migration::remove_deferred_storage::<T>()
-		}
-	}
```

Comment:
note: this comes from the commit: 57402708ebb (Lohann Paterno Coutinho Ferreira 2021-05-03